mobile/src/screens/HomeScreen.tsx

```diff
@@ -12,18 +12,16 @@ const HomeScreen = () => {
   const { userState } = useContext(UserContext);
   const socket = useContext(SocketContext);
 
-  const connectToGame = () => {
+  const moveToLoading = () => {
     socket.emit("assign to room");
-    socket.once("successful assign", (color: string) => {
```

Comment:
If we're removing the handler for this, we need to also cleanup any places it was used in the backend.

Reply:
we used the "successful assign" event on Loading Screen instead so it's still in use